spec/lib/nsa/net_http_tracker_spec.rb

```diff
@@ -48,16 +48,52 @@
             'content-type' => 'text/plain',
             'server' => 'thin'
           )
-        }
+        end
 
-        request_tracker.on_body {
+        request_tracker.on_body do
           expect(request_tracker.response_body).to eql('success')
-        }
-      }
+        end
+      end
 
       response = open("#{base_url}/get_success")
       expect(response.read).to eql("success")
     end
 
+    it 'tracks POST requests' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.